packages/mobile/App/services/sync/utils/preparedInsert.ts

```diff
@@ -0,0 +1,19 @@
+import { Repository } from "typeorm";
+import { DataToPersist } from "../types";
+
+/** 
+ * Much faster than typeorm bulk insert or save
+ * Prepare a raw query and execute it with the values
+ */
+export const preparedInsert = async (repository: Repository<any>, rows: DataToPersist[]) => {
```

Comment:
```suggestion
export const executePreparedInsert = async (repository: Repository<any>, rows: DataToPersist[]) => {
```
or something...initial thought from the title was that this just prepared the prepared insert, but it runs it too (alternative is to make it just prepare the insert and then execute it from the call site)

Reply:
Yeah i thought of this actually! That it was a weird name in that sense